### PR TITLE
[BPK-3748] Bridge flare on iOS

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Unreleased
-
 > Place your changes below this line.
 
+**Added:**
+
+- `bpk-component-flare`:
+  - Added flare component for Android.
```
Comment (shaundon) [inline UNRELEASED.md]:
```suggestion
  - Added flare component.
```

Comment (k0nserv) [reply UNRELEASED.md]: That's what I get for copy and pasting

**Comment by k0nserv** [comment]
> Is the package-lock update expected? Also it's missing docs screenshots :)
I had to run `npm dedupe` because the font reference was failing i.e. `node_modules/bpk-svgs/dist/fonts/BpkIcon.ttf` didn't exists so that's why.

Will add screenshots

**Comment by k0nserv** [comment]
@tiagoengel screenshots added

**Pull request merged**